trompace/mutations/property.py

```diff
@@ -116,7 +84,7 @@ def mutation_add_controlaction_propertyvaluespecification(controlaction_id: str,
         The string for the mutation foradding a control action to a property value specification.
     """
 
-    return mutation_link(controlaction_id, propertyvaluespecification_id, ADD_CONTROLACTION_PROPERTYVALUESPECIFICATION)
+    return format_link_mutation("MergeControlActionAdditionalProperty", controlaction_id, property_id)
```

Comment:
I'll have to check this in more detail, but I think the mutation should be `MergeControlActionObject` - see where I commented in https://github.com/trompamusic/trompa-ce-client/issues/42#issuecomment-665707065 that the last part of the mutation name should include the value of the `field` property in the old mutation. In this case it's `field: object`.
Tomorrow I can check this change with a working CE to see if the process works. Finishing the modelling document for these types will help us clarify these mutations.